packages/web3-common/src/eth_execution_api.ts

```diff
@@ -0,0 +1,266 @@
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/block.json
+	eth_getBlockByHash: (blockHash: HexString32Bytes, hydrated: boolean) => Block;
+	eth_getBlockByNumber: (blockNumber: Uint, hydrated: boolean) => Block;
+	eth_getBlockTransactionCountByHash: (blockHash: HexString32Bytes) => Uint[];
+	eth_getBlockTransactionCountByNumber: (blockNumber: Uint) => Uint[];
+	eth_getUncleCountByBlockHash: (blockHash: HexString32Bytes) => Uint[];
+	eth_getUncleCountByBlockNumber: (blockNumber: Uint) => Uint[];
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/transaction.json
+	eth_getTransactionByHash: (transactionHash: HexString32Bytes) => TransactionInfo;
+	eth_getTransactionByBlockHashAndIndex: (
+		blockHash: HexString32Bytes,
+		transactionIndex: Uint,
+	) => TransactionInfo;
+	eth_getTransactionByBlockNumberAndIndex: (
+		blockNumber: Uint,
+		transactionIndex: Uint,
+	) => TransactionInfo;
+	eth_getTransactionReceipt: (transactionHash: HexString32Bytes) => ReceiptInfo;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/client.json
+	eth_protocolVersion: () => string;
+	eth_syncing: () => SyncingStatus;
+	eth_coinbase: () => Address;
+	eth_accounts: () => Address[];
+	eth_blockNumber: () => Uint;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/execute.json
+	eth_call: (transaction: TransactionWithSender) => HexStringBytes;
+	eth_estimateGas: (transaction: TransactionWithSender) => Uint;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/fee_market.json
+	eth_gasPrice: () => Uint;
+	eth_feeHistory: (
+		blockCount: Uint,
+		newestBlock: BlockNumberOrTag,
+		rewardPercentiles: number[],
+	) => FeeHistoryResult;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/filter.json
+	eth_newFilter: (filter: Filter) => Uint;
+	eth_newBlockFilter: () => Uint;
+	eth_newPendingTransactionFilter: () => Uint;
+	eth_uninstallFilter: (filterIdentifier: Uint) => boolean;
+	eth_getFilterChanges: (filterIdentifier: Uint) => FilterResults;
+	eth_getFilterLogs: (filterIdentifier: Uint) => FilterResults;
+	eth_getLogs: (filter: Filter) => FilterResults;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/mining.json
+	eth_mining: () => boolean;
+	eth_hashrate: () => Uint;
+	eth_getWork: () => [HexString32Bytes, HexString32Bytes, HexString32Bytes];
+	eth_submitWork: (
+		powHash: HexString32Bytes,
+		seedHash: HexString32Bytes,
+		difficulty: HexString32Bytes,
+	) => boolean;
+	eth_submitHashrate: (hashRate: HexString32Bytes, id: HexString32Bytes) => boolean;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/sign.json
+	eth_sign: (address: Address, message: HexStringBytes) => HexString256Bytes;
+	eth_signTransaction: (transaction: TransactionWithSender) => HexStringBytes;
+
+	// https://github.com/ethereum/execution-apis/blob/main/src/eth/state.json
+	eth_getBalance: (address: Address, block: BlockNumberOrTag) => Uint;
+	eth_getStorage: (
```

Comment:
```suggestion
	eth_getStorageAt: (
```

I can't link to the exact doc reference, but it's:

```bash
curl -X POST --data '{"jsonrpc":"2.0", "method": "eth_getStorageAt", "params": ["0x295a70b2de5e3953354a6a8344e616ed314d7251", "0x0", "latest"], "id": 1}' localhost:8545
```

Reply:
The reference to the specification says `eth_getStorage`. So I am not sure if the Eth specification are wrong `eth_getStorageAt` is just an extra alias from the implementation side. 

https://github.com/ethereum/execution-apis/blob/7bcb2f082eed6e786486765b8401dddee6ece339/src/eth/state.json#L29

Reply:
https://github.com/ethereum/execution-apis/issues/95